File: src/Clases/Instruccion/Instruccion.java
package Clases.Instruccion;

import Clases.Entorno.Entorno;

public abstract class Instruccion {
    public String tipoInstruccion;

    public Instruccion(String tipoInstruccion) {
        this.tipoInstruccion = tipoInstruccion;
    }

    public abstract void ejecutar(Entorno entorno);
}

File: Proyecto2/Backend/Clases/Abstractas/Instruccion.ts
import { tipoInstruccion } from "../Utilidades/TipoInstruccion";

export abstract class Instruccion {
    constructor(public linea: number, public columna: number, public tipoInstruccion: tipoInstruccion) {}
    public abstract ejecutar(entorno: any): any;    
}